subsys/net/lib/openthread/platform/shell.c

```diff
@@ -25,8 +25,19 @@ static bool is_shell_initialized;
 
 static int ot_console_cb(void *context, const char *format, va_list arg)
 {
+	char prompt_check[4];
```

Comment:
Nothing wrong on this line, but commit message says:
>  [nrf noup] net: openthread: Temporary fix for OT prompt in the shell 

To me this looks like a disguised `[nrf temphack]` which is no longer allowed.

I think a proper fix is required here.
@carlescufi and @mbolivar-nordic thoughts on this ?

Reply:
A proper fix would be needed on the OpenThread side (to allow to disable CLI prompt), I don't think we have time to cover this. Alternatively, I can propose a similar fix to the upstream Zephyr and make it a `[nrf fromlist]`.

Reply:
I've created an upstream PR with this change and made it a `[nrf fromlist]` commit.